### DISCUSSION TITLE
assxml export

### DISCUSSION THREAD

**Discussion by flugi** [assimp/assimp#4341] [Q&A]
Hello,

Is there a reason why the command line tool does not support assxml reader? I tried to fiddle around small modifications in the result as the timeline processing was a bit off (Unreal exported animation parameter timeline was expanded by the framerate, so frame IDs became seconds) and wanted to test what happens if I edit the assxml tick_cnt parameter to "1", but cannot export.

The main goal will be to use the C++ interface to generate compatible data, I am in the exploration of the data structure and working output data, and the command line app could be a very potent tool in this case.

**Comment by kimkulling** [comment]
Ehrm, no :-). This is a bug in the format detection. Have you tried the latest release?

**Comment by kimkulling** [comment]
Just double-checked that: we do not have a loader for the assimp-xml format. Honestly speaking I am really surprised about that fact.
